applications/launchpad/gui-react/src/containers/WalletContainer/Send/SendModal.tsx

```diff
@@ -0,0 +1,275 @@
+import { useTheme } from 'styled-components'
+import { useForm, Controller, SubmitHandler } from 'react-hook-form'
+
+import Button from '../../../components/Button'
+import Input from '../../../components/Inputs/Input'
+import Modal from '../../../components/Modal'
+import Text from '../../../components/Text'
+
+import t from '../../../locales'
+import { formatAmount } from '../../../utils/Format'
+
+import { SendForm, SendModalProps } from './types'
+import {
+  StyledSendForm,
+  SendFormContent,
+  FormButtons,
+  TagWrapper,
+  TagBox,
+  PleaseWaitContainer,
+  CtaButtonContainer,
+  ResultModal,
+  ResultModalContent,
+  ResultHeader,
+} from './styles'
+import Textarea from '../../../components/Inputs/Textarea'
+import AmountInput from '../../../components/Inputs/AmountInput'
+import SvgTariSignetGradient from '../../../styles/Icons/TariSignetGradient'
+import { useState } from 'react'
+import SvgTBotLoading from '../../../styles/Icons/TBotLoading'
+import Tag from '../../../components/Tag'
+import SvgTBotSearch from '../../../styles/Icons/TBotSearch'
+
+const defaultValues = {
+  amount: 0,
+  address: '',
+  message: '',
+}
+
+/**
+ * Modal with the send transaction form.
+ * @param {boolean} open - is modal open
+ * @param {() => void} onClose - close the modal
+ * @param {number} available - available
+ */
+const SendModal = ({ open, onClose, available }: SendModalProps) => {
+  const theme = useTheme()
+  /**
+   * @TODO - replace with real data
+   */
+  const fee = 0.00043
+
+  /** ---- MOCK SEND PROCESS ------- */
+  const [isProcessing, setIsProcessing] = useState(false)
+  const [result, setResult] = useState<null | 'pending' | 'completing'>(null)
+  /** ---- END MOCK SEND PROCESS ------- */
+
+  const { control, handleSubmit, reset, formState } = useForm<SendForm>({
+    mode: 'onChange',
+    defaultValues,
+  })
+
+  const cancel = () => {
+    reset(defaultValues)
+    setIsProcessing(false)
+    setResult(null)
+    onClose()
+  }
+
+  const onSubmitForm: SubmitHandler<SendForm> = _ => {
+    reset(defaultValues)
+    setIsProcessing(true)
+  }
+
+  const validateAmount = (amount: number) => {
+    if (amount > available + fee) {
+      return t.wallet.transaction.errors.exceedsAvailableAndFee
+    }
+
+    if (amount === 0) {
+      return ''
```

Comment:
I'm not sure, is there a difference in behaviour between returning `''` and `undefined` ?